bot/cogs/bot.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
         # Stores allowed channels plus unix timestamp from last call
-        self.previous_format_times = {HELP1_CHANNEL: 0,
+        self.previous_hint_times = {HELP1_CHANNEL: 0,
```

Comment:
I still hate this variable name. what hint? we don't really use the word hint in any variable names or comments. how about `self.channel_cooldowns` or something so we're consistent. Also I'm not sure about that, and if you can come up with something better then use that instead.